systemd/systemd.go

```diff
@@ -437,12 +437,12 @@ func MountUnitPath(baseDir string) string {
 }
 
 func (s *systemd) WriteMountUnitFile(name, what, where, fstype string) (string, error) {
-	extra := ""
+	options := []string{"nodev"}
```

Comment:
the basic template btw has this:

/dev/ r,
/dev/**/ r,

so we are probably confused about something

either way we can probably use /dev/{u,}random to write a test that access to that still works